src/Tickets/Commerce/Shortcodes/Checkout_Shortcode.php

```diff
@@ -115,10 +108,21 @@ public function get_gateways_active() {
 	 * @return int The number of connected gateways.
 	 */
 	public function get_gateways_connected() {
+		return $this->get_gateways_available();
```

Comment:
@rafsuntaskin please keep the old code of gateways connected here. We handle the available, but we may use the number of connected gateways as well. So we should keep the logic of the connected gateways in place.